### PR TITLE
Symantec Proxy SG Parser Update

### PR DESCRIPTION
**Pull request by elforb**
Fixes #
  - Update regex in parser according to feedback in #2042

### CODE REVIEW

**Comment by v-jayakal** [comment]
@shainw, an issue has been raised on Symantec Parser https://github.com/Azure/Azure-Sentinel/issues/2042. Appreciate your help if you can take a look at this, Thanks.

**Review by appelboom — APPROVED** [review]

File: Parsers/SymantecProxySG/SymantecProxySG.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@
 Syslog
 | where Computer in ("datasource") and Facility == "local0"
```
Comment (v-jayakal) [inline Parsers/SymantecProxySG/SymantecProxySG.txt:30]: @appelboom, should it not include changes here? 
ParserChanges

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: Parsers/SymantecProxySG/SymantecProxySG.txt (hunk 1 above)
Comment (appelboom) [reply Parsers/SymantecProxySG/SymantecProxySG.txt:30]: Hi, No. They are fields the customer must change. Described in first lines.

**Review by appelboom — APPROVED** [review]

**Comment by v-jayakal** [comment]
Closing as this is for different datetime format.

**Pull request closed without merging**